development/prow-installer/cmd/installer/main.go

```diff
@@ -23,18 +22,67 @@ func main() {
 	}
 
 	if *credentialsfile == "" {
-		log.Fatalf("Missing required argument : -credentialsfile")
+		log.Fatalf("Missing required argument : -credentials-file")
 	}
 
-	var InstallerConfig installer.InstallerConfig
-	InstallerConfig.ReadConfig(*config)
+	var Config installer.Config
+	if err := Config.ReadConfig(*config); err != nil {
+		log.Fatalf("Error reading config file %v", err)
+	}
+	//Config.Labels["created-on"] = time.Now()
+	ctx := context.Background()
 
-	AccessManager := accessmanager.NewAccessManager(*credentialsfile)
+	storageConfig := &storage.Option{
+		ProjectID:      Config.Project,
```

Comment:
You see here, that I don't know easily now if `Config.Project` is an exported constant or a field of the config variable above. With `config.Project` it would be more clear.